x/tokenregistry/types/tx.pb.go

```diff
@@ -204,37 +204,339 @@ func (m *MsgDeregisterResponse) XXX_DiscardUnknown() {
 
 var xxx_messageInfo_MsgDeregisterResponse proto.InternalMessageInfo
 
+type TokenMetadataAddRequest struct {
+	CosmosSender string         `protobuf:"bytes,1,opt,name=cosmos_sender,json=cosmosSender,proto3" json:"cosmos_sender,omitempty"`
+	Metadata     *TokenMetadata `protobuf:"bytes,2,opt,name=metadata,proto3" json:"metadata,omitempty" yaml:"metadata"`
+}
+
+func (m *TokenMetadataAddRequest) Reset()         { *m = TokenMetadataAddRequest{} }
+func (m *TokenMetadataAddRequest) String() string { return proto.CompactTextString(m) }
+func (*TokenMetadataAddRequest) ProtoMessage()    {}
+func (*TokenMetadataAddRequest) Descriptor() ([]byte, []int) {
+	return fileDescriptor_d09312f3deb69cfe, []int{4}
+}
+func (m *TokenMetadataAddRequest) XXX_Unmarshal(b []byte) error {
```

Comment:
Do we need those 'XXX' prefixes? Usually, they mean 'FIXME', 'TODO', 'HACK', or something like that.
If we can, it would be great to remove those prefixes or replace them with something more descriptive.

Reply:
FYI, that's generated code.  They're part of the protobuf system we're using - https://stackoverflow.com/questions/50704319/xxx-type-in-generated-pb-go-file